### PR TITLE
fix: change filter for lookup

### PR DESCRIPTION
**Pull request by gene-redpanda**
Fixes an issue with the lookup filter caused by a change in the version of the instances available to AWS. 

I would very much like to get rid of this lookup table but I can't work out a way to do it that doesn't break existing behavior without introducing something worse. 

I will probably just do a breaking change at some point (the fail state for an invalid input is an error in the lookup, no destruction and easily resolved) but not today 😅

### CODE REVIEW

File: .buildkite/pipeline.yaml [hunk 1]
```diff
@@ -113,10 +113,10 @@ steps:
       - seek-oss/aws-sm#v2.3.2:
           json-to-env:
             - json-key: .
-              secret-id: sdlc/prod/buildkite/devex_gcp_creds_base64
```
Comment (andrewstucki) [inline .buildkite/pipeline.yaml:116]: Are these actually different creds or did the secret just get moved?

File: examples/public-vpc-is4/main.tf [hunk 2]
```diff
@@ -58,13 +58,13 @@ module "redpanda-cluster" {
   distro                 = var.distro
   hosts_file             = var.hosts_file
   tags                   = var.tags
-  subnets                = {
+  subnets = {
```
Comment (andrewstucki) [inline examples/public-vpc-is4/main.tf:61]: was this `terraform fmt`'d?

File: examples/simple-fedora/main.tf [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      version = "~> 3.5"
+    }
+    time = {
+      source  = "hashicorp/time"
+      version = "~> 0.9"
+    }
+  }
+}
+
+variable "region" {
```
Comment (andrewstucki) [inline examples/simple-fedora/main.tf:37]: This should also get passed in to the `simple-example` module block as `aws_region`, no? (though it also looks like it defaults to `us-west-2`)

**Review by andrewstucki — APPROVED** [review]
Generally looks fine to me, though I think you may want to pass `aws_region` into your module block just for consistency.

File: .buildkite/pipeline.yaml (hunk 1 above)
Comment (gene-redpanda) [reply .buildkite/pipeline.yaml:116]: Secrets got moved

**Review by r-vasquez — APPROVED** [review]

**Pull request merged**